grabbit/build.gradle

```diff
@@ -70,6 +59,7 @@ jar.manifest {
     instruction 'Import-Package', 'org.springframework.aop.scope; version="[3.1,4.0)"'
     instruction 'Import-Package', 'org.springframework.scheduling.concurrent; version="[3.1,4.0)"'
     instruction 'Import-Package', "org.aopalliance.aop;version=1.0.0"
+    instruction 'Import-Package', "sun.misc"
```

Comment:
I am not sure if `sun.misc` is required by our bundle. I thought `sun.misc` is only required by `groovy's json package` ?